src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -113,6 +113,14 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
           clusterMonitorSupervisor ! ClusterCreated(savedCluster)
           savedCluster
         }
+        clusterFuture.onComplete {
```

Comment:
`onFailure` is deprecated :(

Reply:
Can do `ClusterFuture.failed.foreach { throwable => ... }`

Reply:
Er maybe not because it might be successful